pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -0,0 +1,223 @@
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:data", "data")
+#@ load("@ytt:ip", "ip")
+#@ load("lib/helpers.star", "get_tkr_version_from_tkr_name")
+
+#@ vsphereControlPlaneEndpointAsCidr = ""
+#@ if data.values.VSPHERE_CONTROL_PLANE_ENDPOINT:
+#@   vsphereControlPlaneEndpoint, err = assert.try_to(lambda : ip.parse_addr(data.values.VSPHERE_CONTROL_PLANE_ENDPOINT))
+#@   if err == None:
+#@     if vsphereControlPlaneEndpoint.is_ipv4():
+#@       vsphereControlPlaneEndpointAsCidr = vsphereControlPlaneEndpoint.string() + "/32"
+#@     end
+#@     if vsphereControlPlaneEndpoint.is_ipv6():
+#@       vsphereControlPlaneEndpointAsCidr = vsphereControlPlaneEndpoint.string() + "/128"
+#@     end
+#@   end
+#@ end
+
+#@ def antrea_configs_exist():
+#@   return (data.values.NSXT_POD_ROUTING_ENABLED or
+#@           data.values.ANTREA_NO_SNAT or
+#@           data.values.ANTREA_TRAFFIC_ENCAP_MODE != "encap" or
+#@           not data.values.ANTREA_PROXY or
+#@           not data.values.ANTREA_ENDPOINTSLICE or
+#@           not data.values.ANTREA_POLICY or
+#@           not data.values.ANTREA_NODEPORTLOCAL or
+#@           not data.values.ANTREA_TRACEFLOW or
+#@           not data.values.ANTREA_EGRESS or
+#@           data.values.ANTREA_FLOWEXPORTER or
+#@           data.values.ANTREA_DISABLE_UDP_TUNNEL_OFFLOAD)
+#@ end
+
+#@ def vspherecpi_configs_exist():
+#@   return data.values.PROVIDER_TYPE == "vsphere" and (data.values.VSPHERE_SERVER or data.values.NSXT_MANAGER_HOST)
+#@ end
+
+#@ def vspherecsi_configs_exist():
+#@   return data.values.PROVIDER_TYPE == "vsphere" and (not data.values.VSPHERE_INSECURE or
+#@           data.values.USE_TOPOLOGY_CATEGORIES)
+#@ end
+
+#@ def should_create_clusterbootstrap():
+#@   return data.values.CNI != "antrea" or antrea_configs_exist() or vspherecsi_configs_exist() or vspherecpi_configs_exist()
+#@ end
+
+#@ if antrea_configs_exist():
+---
+apiVersion: cni.tanzu.vmware.com/v1alpha1
+kind: AntreaConfig
+metadata:
+  name: #@ data.values.CLUSTER_NAME
+  namespace: #@ data.values.NAMESPACE
+spec:
+  antrea:
+    config:
+      #@ if data.values.NSXT_POD_ROUTING_ENABLED:
```

Comment:
@lubronzhan could you review antrea pod routing?

Reply:
It's jsut copied over from here
https://github.com/vmware-tanzu/tanzu-framework/blob/ac703899beb7d2aae78955ed06ac66382e4cf079/pkg/v1/providers/ytt/02_addons/cni/antrea/antrea_addon_data.lib.yaml#L71-L85?
Looks the same to me